server/guiserver/manage.py

```diff
@@ -161,6 +161,7 @@ def setup():
         'gzip', type=bool, default=False,
         help='Enable gzip compression in the gui.')
     define('gtm', type=bool, default=False, help='Enable Google tag manager.')
+    define('gisf', type=bool, default=False, help='Enable GUI in store front.')
```

Comment:
Why did this cause the charm to not deploy? If this is so critical shouldn't we have a test for it?